### PR TITLE
Added a new net which performs the standard dropout rescaling

### PR DESCRIPTION
**Pull request by garyxcheng**

### CODE REVIEW

**Comment by garyxcheng** [comment]
@googlebot I fixed it.

File: shrink_unshrink/models.py [hunk 1]
```diff
@@ -223,6 +223,86 @@ def create_conv_dropout_model(conv1_filters=32,
...
+  ```
+  For `only_digits=False`, the last dense layer is slightly larger.
+
+  Args:
+    conv1_filters: The number of convolutional filters in the 1st convolutional
+      layer
+    conv2_filters: The number of convolutional filters in the 2nd convolutional
+      layer
+    dense_size: The number of neurons in the last dense layer
+    only_digits: If `True`, uses a final layer with 10 outputs, for use with the
```
Comment (jkr26) [inline shrink_unshrink/models.py:277]: Let's add the new parameters to the arg section, along with descriptions, since this might be a little subtle

Comment (garyxcheng) [reply shrink_unshrink/models.py:277]: The new commit does add some description

**Comment by garyxcheng** [comment]
@googlebot I fixed it.

File: shrink_unshrink/models.py (hunk 1 above)
Comment (jkr26) [reply shrink_unshrink/models.py:277]: thanks! Not used to reviewing code on GH, I thought the first commit was the lasts commit...

**Review by jkr26 — APPROVED** [review]

**Pull request closed without merging**